src/Sylius/Behat/Resources/config/services/contexts/api/shop.xml

```diff
@@ -69,7 +69,8 @@
             <argument type="service" id="sylius.product_variant_resolver.default" />
             <argument type="service" id="api_platform.iri_converter" />
             <argument type="service" id="sylius.behat.shared_storage" />
-            <argument type="service" id="sylius.product_variant_resolver.default" />
```

Comment:
FYI: some leftover - removed it because it was not used and btw it is declared in line 69